pkg/server/home_workspaces.go

```diff
@@ -272,7 +272,7 @@ func (h *homeWorkspaceHandler) ServeHTTP(rw http.ResponseWriter, req *http.Reque
 	logger := klog.FromContext(ctx)
 	lcluster, err := request.ValidClusterFrom(ctx)
 	if err != nil {
-		responsewriters.InternalError(rw, req, err)
+		h.apiHandler.ServeHTTP(rw, req)
```

Comment:
it looks like the `home-ws` is first in the chain, capturing `healthz/readyz` as well.